bears/general/SpaceConsistencyBear.py

```diff
@@ -14,7 +14,9 @@ def run(self,
             tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
             enforce_newline_at_EOF: bool=True):
         '''
-        Checks the space consistency for each line.
+        Check and correct spacing for all textual data. This includes usage of
+        tabs vs. spaces, trailing whitespace and (missing) newlines before
+        the end of the file.
```

Comment:
well i agree with you, but he provided a link @ https://github.com/coala-analyzer/coala-bears/pull/333/commits/10ed0de7dfc1dcb44390696ebc4aca47423c2606 , that's why i wanted to keep it consistent. should we have link on both, or neither imo